lucene/queryparser/src/java/org/apache/lucene/queryparser/surround/parser/QueryParserTokenManager.java

```diff
@@ -68,7 +68,7 @@ private int jjMoveStringLiteralDfa0_1(){
 private int jjMoveNfa_1(int startState, int curPos)
 {
    int startsAt = 0;
-   jjnewStateCnt = 38;
+   jjnewStateCnt = 40;
```

Comment:
Just curious, is this whole file generated, or do we manually have to edit what looks to me like black magic? 😅

Reply:
whole file is generated, yes 😅